tests/test_inspector.py

```diff
@@ -27,19 +28,22 @@
 from nwbinspector.utils import FilePathType, is_module_installed
 from nwbinspector.tools import make_minimal_nwbfile
 
-
-try:
-    with NWBHDF5IO(
-        path="https://dandiarchive.s3.amazonaws.com/blobs/11e/c89/11ec8933-1456-4942-922b-94e5878bb991",
-        mode="r",
-        load_namespaces=True,
-        driver="ros3",
-    ) as io:
-        nwbfile = io.read()
-    HAVE_ROS3 = True
-except ValueError:  # ValueError: h5py was built without ROS3 support, can't use ros3 driver
-    HAVE_ROS3 = False
+DANDI_TESTS_NONETWORK = os.environ.get("DANDI_TESTS_NONETWORK", "")
+NO_NETWORK = strtobool(DANDI_TESTS_NONETWORK) if DANDI_TESTS_NONETWORK != "" else False
+if not NO_NETWORK:
+    try:
+        with NWBHDF5IO(
+            path="https://dandiarchive.s3.amazonaws.com/blobs/11e/c89/11ec8933-1456-4942-922b-94e5878bb991",
+            mode="r",
+            load_namespaces=True,
+            driver="ros3",
+        ) as io:
+            nwbfile = io.read()
+        HAVE_ROS3 = True
+    except ValueError:  # ValueError: h5py was built without ROS3 support, can't use ros3 driver
+        HAVE_ROS3 = False
```

Comment:
Rather than reading a file here, you could just check whether internet access is available and whether the ros3 driver is installed. Here is how PyNWB does this in the setup for the ROS3 tests:

```
    def setUp(self):
        # Skip ROS3 tests if internet is not available or the ROS3 driver is not installed
        try:
            urllib.request.urlopen('https://dandiarchive.s3.amazonaws.com/ros3test.nwb', timeout=1)
        except urllib.request.URLError:
            self.skipTest("Internet access to DANDI failed. Skipping all Ros3 streaming tests.")
        if 'ros3' not in h5py.registered_drivers():
            self.skipTest("ROS3 driver not installed. Skipping all Ros3 streaming tests.")
```

See https://github.com/NeurodataWithoutBorders/pynwb/blob/92ed81bcdc7f5243f08a72fd5ea10c0d5a0c5b90/tests/integration/ros3/test_ros3.py#L13-L20

Reply:
Ahh, that's handy. Thanks!